app/Console/Commands/MakeDefaultURLsNull.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class MakeDefaultURLsNull extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:nullurl';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Change "default" URLs in the Posts table to null';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:nullurl: Starting!');
+
+        // Get all posts with "default" for the url
+        $posts = Post::where('url', 'default')->get();
```

Comment:
This might be something to handle [chunked query or cursor](https://laravel.com/docs/5.5/queries#chunking-results) (depending on how many `default` posts there are).

Reply:
I thought about that too, there are ~60k. Worth chunking?

Reply:
Mmm yeah, I think so. That's a lot of stuff to cram into memory otherwise.